orchestrator/nodes/cron/cron.html

```diff
@@ -0,0 +1,184 @@
+<script type="text/javascript">
+    RED.nodes.registerType('cron', {
+        // (string) the palette category the node appears in
+        category: "function",
+        // (string) the icon to use
+        icon: "timer.png",
+        // (string) the background colour to use
+        color: "#71c4b1",
+        // (number) how many inputs the node has, either 0 or 1
+        inputs: 1,
+        // (number) how many outputs the node has. Can be 0 or more.
+        outputs: 1,
+        inputLabels: "in",
+        outputLabels: ["out"],
+        // (object) the editable properties for the node
+        defaults: {
+            name: {
+                value: "",
+                required: false
+            },
+            operation: {
+                value: "CREATE",
+                required: true
+            },
+            cronTimeExpression: {
+                value: "",
+                validate: function(v) {
+                    if($("#node-input-operation option:selected").val() === 'CREATE') {
+                        let cronregex = new RegExp(/^(\*|([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])|\*\/([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])) (\*|([0-9]|1[0-9]|2[0-3])|\*\/([0-9]|1[0-9]|2[0-3])) (\*|([1-9]|1[0-9]|2[0-9]|3[0-1])|\*\/([1-9]|1[0-9]|2[0-9]|3[0-1])) (\*|([1-9]|1[0-2])|\*\/([1-9]|1[0-2])) (\*|([0-6])|\*\/([0-6]))$/);
```

Comment:
This regex seems don't follow the field specification:
"The cron time string is five values separated by spaces. From the most left value to the righest one, we have minute (0-59), hour (0-23), day of month (1-31), month (1-12 or names), and day of week (0-7 or names, 0 or 7 are sunday). You may use multiple values separated by comma e.g 0,10,20,30,40,50, * * * * ."
The multiple values are not supported